jans-auth-server/server/src/main/java/io/jans/as/server/authorize/ws/rs/AuthorizeRestWebServiceImpl.java

```diff
@@ -782,16 +790,37 @@ private Response requestAuthorization(
     }
 
     @Nullable
-    private ResponseMode extractResponseMode(String request) {
-        final Jwt jwt = Jwt.parseSilently(request);
-        if (jwt == null) {
+    private JsonWebResponse parseRequestToJweJwt(String request) {
+        String[] parts = request.split("\\.");
```

Comment:
Before we had `Jwt.parseSilently(request);` which was nullity safe. Now without null check we will get NPE here for all not `request` test cases. Did you run tests against these changes?

Reply:
I executed only the JARM testd but not with any tests where request is null. Should I check if (request!=null) before split.